setup/quickstart/faq.md

```diff
@@ -165,3 +165,36 @@ For example,
 ```bash
 DEFAULT_JVM_OPTS=-Dhttp.proxyHost=my.proxy.domain.com -Dhttp.proxyPort=3128
 ```
+
+## I want to run a Spinnaker service (Clouddriver, Echo, etc) behind an HTTP proxy server
+
+For most Spinnaker service communication, this can be accomplished by setting appropriate 
+JVM options for the service you want to proxy. For example, if you wanted to proxy Echo
+communication for Slack notifications, you would add the following proxy settings to 
+`~/.hal/default/service-settings/echo.yml`
+
+```yaml
+env:
+  JAVA_OPTS: "-XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap -XX:MaxRAMFraction=2
+              -Dhttp.proxyHost=<proxy host> -Dhttp.proxyPort=<proxy port> -Dhttps.proxyHost=<proxy host>
+              -Dhttps.proxyPort=<proxy port> -Dhttp.nonProxyHosts='localhost|127.*|[::1]|*.spinnaker'"
+```
+
+These settings will forward all external communication through the proxy server specified while
+keeping internal traffic non-proxied. Additional information can be found 
+[here.](https://docs.oracle.com/javase/8/docs/technotes/guides/net/proxies.html){:target="\_blank"}
+
+The Kubernetes V2 provider in the Clouddriver must be handled differently. Because the 
```

Comment:
Delete "in the Clouddriver"

"The Kubernetes V2 provider must be handled differently."

Reply:
@dorbin - This change has been made as well. Thank you